website/docs/r/cse_insights_configuration.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_insights_configuration"
+description: |-
+  Provides a CSE Insights Configuration
+---
+
+# sumologic_cse_insights_configuration
+Provides a CSE Insights Configuration.
```

Comment:
We should maybe say "Provides the CSE Insights Configuration for the whole organization. There can be only one configuration per organization." or something like that.

Reply:
done under:
https://github.com/SumoLogic/terraform-provider-sumologic/pull/274/commits/22ae5d0e51f44dc94e2ed6cb28653422ce2ef257